src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -264,11 +265,9 @@ protected virtual void Initialize()
                     TraceWriter = new ConsoleTraceWriter(hostTraceLevel);
                 }
 
-                _debugModeFileWatcher = new FileSystemWatcher(hostLogPath, ScriptConstants.DebugSentinelFileName)
-                {
-                    EnableRaisingEvents = true
-                };
-                _debugModeFileWatcher.Created += OnDebugModeFileChanged;
```

Comment:
The `Changed` subscription remains in place (line 271) as it is now the only event you need to subscribe to.